toolkit/tools/internal/testutils/testrpms/extract_packages.py

```
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+"""
+Extract package list from YAML configuration files.
+"""
+
+import yaml
+import sys
+import argparse
+import os
+
+def extract_packages_from_file(config_file_path):
+    """Extract packages from a single YAML file."""
+    try:
+        with open(config_file_path, 'r') as f:
+            data = yaml.safe_load(f)
+            
+            # Handle direct package list (like in golang.yaml)
+            if 'packages' in data and data['packages']:
+                return data['packages']
+            
+            packages = data.get('os', {}).get('packages', {}).get('install', [])
+            return packages
+            
+    except FileNotFoundError:
```

Is there a reason why you are handling these exceptions? Why not just let the script fail?

---

updated the script to fail